From 0f3827702bfc211da179807dc2585c244c9af4ca Mon Sep 17 00:00:00 2001
From: Josh Jensen <josh@everrush.com>
Date: Sat, 15 Jun 2024 12:21:18 -0600
Subject: Prevent Android from destroying the surface that bgfx owns

---
 src/core/android/SDL_android.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/core/android/SDL_android.c b/src/core/android/SDL_android.c
index 60620d8..e4c9236 100644
--- a/src/core/android/SDL_android.c
+++ b/src/core/android/SDL_android.c
@@ -1100,8 +1100,8 @@ retry:
 
 #ifdef SDL_VIDEO_OPENGL_EGL
         if (data->egl_surface != EGL_NO_SURFACE) {
-            SDL_EGL_DestroySurface(_this, data->egl_surface);
-            data->egl_surface = EGL_NO_SURFACE;
+            //SDL_EGL_DestroySurface(_this, data->egl_surface);
+            //data->egl_surface = EGL_NO_SURFACE;
         }
 #endif
 
-- 
2.45.1.windows.1

